include/ccf/service/tables/service.h

```diff
@@ -39,6 +39,8 @@ namespace ccf
     nlohmann::json service_data = nullptr;
     /// TxID at which current service was created
     std::optional<ccf::TxID> current_service_create_txid = std::nullopt;
+    /// Size of recent_cose_proposals window
+    std::optional<size_t> recent_cose_proposals_window_size = std::nullopt;
```

Comment:
It seems that this would fit better in the [service configuration table](https://microsoft.github.io/CCF/main/audit/builtin_maps.html#service-config)?

Reply:
By updating the `cchost_config.json` schema, it would also be settable by the operator at startup (see https://microsoft.github.io/CCF/main/operations/configuration.html#service-configuration).

Reply:
Thank you for the suggestion, it definitely looks like a better fit, I'll move it there.